src/bootstrap-fork.js

```diff
@@ -172,18 +169,4 @@ function terminateWhenParentTerminates() {
 	}
 }
 
-function configureCrashReporter() {
```

Comment:
@deepak1556 will Electron 9 enable the crash reporter for each child process automatically? How in AppCenter would we be able to tell apart where the crash is coming from?

Reply:
Yup it will automatically watch all the child processes including the ones launched by node fork.

> How in AppCenter would we be able to tell apart where the crash is coming from?

we haven't been distinguishing them currently in appcenter, am I missing something here ?

Reply:
Yes you are right, though it would be helpful if there was a way I guess. I think we can already tell apart renderer from main process, no? Would be great to see the executable name appear as metadata somehow.

Reply:
we are sending the executable name currently but appcenter isn't displaying them in anyway :(

Reply:
A little correction, for linux we need to explicitly start the crash reporter for child processes because of breakpad, have accommodated for it.